decidim-core/lib/decidim/core/engine.rb

```diff
@@ -32,6 +32,7 @@
 require "premailer/rails"
 require "nokogiri"
 require "geocoder"
+require "letter_opener_web"
```

Comment:
@josepjaume @deivid-rodriguez Fixed it! https://github.com/decidim/decidim/pull/1527/commits/0a864d35c0b840f2864167da687455cb79de7571